calcom/cal.com
# feat: Slack Icon changed to white in dark mode

## What does this PR do?
I updated the Slack icon to white color in dark mode on the 404 Page.

Fixes #9210

## Screenshot
![Screenshot 2023-05-31 at 12 27 20 AM](https://github.com/calcom/cal.com/assets/44976328/ea08ef1e-b007-4118-8dcd-961ec0996941)


## Type of change
- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
Visit a page that does not exist to render a 404 Page and you can see that the slack icon is now white in color.
